README.md

```diff
@@ -95,7 +97,7 @@ formats.
 
 ### Code Review Requests
 
-Code review requests are stored in the "refs/notes/devtools/reviews" ref, and
+Code review requests are stored in the "refs/notes/git-review/requests" ref, and
```

Comment:
That's a good point; only part of the review is stored in that ref, so it should have been more specific. I favor long names over short ones, so I would change that to "review-requests" rather than just "requests", but still it's a good point.

Unfortunately, it's too late to change that for this version of the schema (it's a backwards-incompatible change), but if we ever have to make a backwards-incompatible change, then we should keep that in mind. That being said; I don't have any reason to believe that we will make such a change.